server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts

```diff
@@ -713,7 +713,14 @@ export class SummaryWriter implements ISummaryWriter {
 		let logTail: ISequencedDocumentMessage[] = [];
 
 		if (this.getDeltasViaAlfred) {
-			logTail = await this.deltaService.getDeltas("", this.tenantId, this.documentId, gt, lt);
+			logTail = await this.deltaService.getDeltas(
+				"",
+				this.tenantId,
+				this.documentId,
+				gt,
+				lt,
```

Comment:
We may need to do a check here, if `gt` and `lt` range is larger than `getDeltasRequestMaxOpsRange`, we break the request into chunks with max range of `getDeltasRequestMaxOpsRange`

Reply:
good point, I updated the logic for that in deltaManager.ts